pkg/windows/windows.go

```diff
@@ -256,23 +248,42 @@ type windows struct {
 	log      logr.Logger
 	// defaultShellPowerShell indicates if the default SSH shell is PowerShell
 	defaultShellPowerShell bool
+	// filesToTransfer is the list of files needed for the windows VM
```

Comment:
This is not a list but a map. Please call out what the key value definitions are here.